### DISCUSSION TITLE
System.AccessViolationException with csvwrite

### DISCUSSION THREAD

**Discussion by Johntron1219** [bonsai-rx/bonsai#1080] [Q&A]
Hello, 
I've written a program that tracks the movement of a mouse's paws, and records when they are in a certain place. The program is admittedly a lot of work for the computer to run, and so I have been trying to run it in sections. One section is here: 

![working sample](https://user-images.githubusercontent.com/99308415/195419233-2205ccf3-d585-4401-9d5e-ad6825056720.PNG)
This workflow will run without exception. 

However, when I run this workflow: 
![notworkingsample](https://user-images.githubusercontent.com/99308415/195419370-ffbd85a8-8866-4b62-a058-6d42aca6c131.PNG)

It throws the below exception: 

![IMG_3977](https://user-images.githubusercontent.com/99308415/195418776-50cb1da9-f2a7-4c58-a8e2-85bbbc302344.JPG)
 
Any idea as to what to change/fix? 

Thanks!

John

**Comment by glopesdev** [comment]
This seems like an instance of a known issue of Bonsai 2.x opening .mp4 files. Are the movie files in MP4 format?

If so, this is already corrected for the new Bonsai 2.7 which is coming out soon. If you want you can try the preview at: https://github.com/bonsai-rx/bonsai/releases/tag/2.7-rc5
